assets/scss/_custom.scss

```diff
@@ -354,12 +354,77 @@ main {
     word-break: break-word;
   }
 
+/* SCSS Related to the Metrics Table */
+
+// Define the variable for the table caption
+$table-caption: "This is the list of STABLE metrics emitted from core Kubernetes components";
```

Comment:
This doesn't support localization. Content should not be part of the style sheet.